frontend/src/main/scala/bloop/engine/tasks/CompilationTask.scala

```diff
@@ -159,8 +158,9 @@ object CompilationTask {
         dependentResults.collect { case (p, Compiler.Result.NotOk(_)) => p }
       if (!failedDependentProjects.isEmpty) {
         val failedProjects = failedDependentProjects.map(p => s"'${p.name}'").mkString(", ")
-        logger.warn(
-          s"Skipping compilation of project '$project'; dependent $failedProjects failed to compile.")
+        // FIXME: What to print?!?
```

Comment:
What you can do here is `Dag.directDependencies(dag)` which will return you a list of projects, and you can print the list.
  
I might rework this whenever this PR is ready in an independent commit and remove this part of the code -- I need to give it more thought. But for now, doing this is just fine.